### PR TITLE
Throw Api exceptions

### PR DESCRIPTION
**Pull request by brieucpouliquen**

### CODE REVIEW

**Comment by brieucpouliquen** [comment]
mergeable

**Comment by jbcrestot** [comment]
rebase plz

File: src/ScrumBoardItBundle/Controller/DefaultController.php [hunk 1]
```diff
@@ -129,8 +129,7 @@ public function addFlagAction(Request $request)
             $apiService = $this->get($this->getUser()->getApi());
             $selected = $request->request->get('issues');
             $apiService->addFlag($request, $selected);
-        } catch(\Exception $e) {
-            
+        } catch (\Exception $e) {
```
Comment (jbcrestot) [inline src/ScrumBoardItBundle/Controller/DefaultController.php]: Remove this or log exception, it stifles all exceptions. It's best way to lose time in futur to find why you don't see exception

File: src/ScrumBoardItBundle/Resources/views/Default/index.html.twig [hunk 2]
```diff
@@ -60,6 +60,9 @@
         <div class="row">
             {% if not issues|length %}
             <div class="col-md-12">
+                {% if error %}
+					<div class="alert alert-danger">{{ error.message|trans({},'messages') }}</div>
```
Comment (jbcrestot) [inline src/ScrumBoardItBundle/Resources/views/Default/index.html.twig]: tabulation?

File: src/ScrumBoardItBundle/Security/GitHubAuthenticator.php [hunk 3]
```diff
@@ -36,9 +36,13 @@ public function checkCredentials($credentials, UserInterface $user)
         $user->setHash("$login:$password");
 
         $url = $this->config['host'].'user';
-        $results = $this->apiCaller->call($user, $url);
+        try {
+            $results = $this->apiCaller->call($user, $url);
+        } catch (\Exception $e) {
+            return false;
```
Comment (jbcrestot) [inline src/ScrumBoardItBundle/Security/GitHubAuthenticator.php]: Can you explain the use case?

Comment (brieucpouliquen) [inline src/ScrumBoardItBundle/Security/GitHubAuthenticator.php]: return false to redirect to the login form with an authentication error.

**Pull request merged**